### ISSUE TITLE
SplitTooLongLine output for multiline [Tags] violates SPC15

### ISSUE THREAD

**Issue by maciej-ociepa** [MarketSquare/robotframework-robocop#1794]
### Description

Starting with Robocop 8.4.0, `robocop format` can produce multiline
`[Tags]` settings that are subsequently rejected by `robocop check`
with SPC15.

The file passes SPC15 before formatting but fails after applying the
formatter.

### Steps to reproduce

Create `example.robot`:

```robotframework
*** Settings ***
Documentation    Minimal formatter/linter reproduction.


*** Test Cases ***
Example
    [Documentation]    Example test.
    [Tags]
    ...    first-tag
    ...    second-tag
    ...    third-tag

    No Operation
```

Run:

```shell
robocop check example.robot --select SPC15
robocop format example.robot --line-length 40 --no-cache
robocop check example.robot --select SPC15
```

### Actual behavior

The first check reports no issues.

The formatter changes the tags to:

```robotframework
    [Tags]    first-tag
    ...    second-tag
    ...    third-tag
```

The second check reports:

```text
SPC15 Continuation line is not aligned with the previous one
```

### Expected behavior

Output produced by `robocop format` should pass `robocop check`.

The formatter could either preserve the original layout:

```robotframework
    [Tags]
    ...    first-tag
    ...    second-tag
    ...    third-tag
```

or align the continuation values with the first tag:

```robotframework
    [Tags]    first-tag
    ...       second-tag
    ...       third-tag
```

### Regression information

The behavior was tested with the following versions:

- 8.3.2: the formatter leaves the original layout unchanged and SPC15 passes
- 8.4.1: the formatter changes the layout and SPC15 fails
- 8.5.0: the formatter changes the layout and SPC15 fails

The regression appears to have been introduced in 8.4.0 by #1784
(`keep first setting argument on the setting line in SplitTooLongLine`).

That change intentionally keeps the first argument of settings such as
`[Arguments]` and `[Tags]` on the setting line. For multiline `[Tags]`,
however, the continuation indentation produced by the formatter conflicts
with SPC15.

### Environment

- Robocop: 8.4.0 and later
- Robot Framework: 7.4.2
- Python: 3.12
- OS: Linux

**Comment by maciej-ociepa** [comment]
Closing, it has been resolved by https://github.com/MarketSquare/robotframework-robocop/pull/1796
